portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -47,7 +47,7 @@ definition {
 
 		FormFields.enableSwitch(fieldName = "Required Field");
 
-		DataEngine.waitForAutoSave();
```

Comment:
The use of the macro DataEngine.waitForAutoSave(); is not necessary.
In the macro, If Publish button is present, Click and right after, the macro Form.publishForm(); is called but does not find the button that was already clicked in the first macro.